packages/app-store/basecamp3/api/add.ts

```diff
@@ -0,0 +1,35 @@
+import type { NextApiRequest } from "next";
+import { stringify } from "querystring";
+
+import { WEBAPP_URL } from "@calcom/lib/constants";
+import { defaultHandler, defaultResponder } from "@calcom/lib/server";
+import prisma from "@calcom/prisma";
+
+import { getBasecampKeys } from "../lib/getBasecampKeys";
+
+async function handler(req: NextApiRequest) {
+  await prisma.user.findFirstOrThrow({
+    where: {
+      id: req.session?.user?.id,
+    },
+    select: {
+      id: true,
+    },
+  });
+
+  const { client_id } = await getBasecampKeys();
+
+  const params = {
+    type: "web_server",
+    client_id,
+  };
+  const query = stringify(params);
+  const url = `https://launchpad.37signals.com/authorization/new?${query}&redirect_uri=${
```

Comment:
Is this suppose to be hardcoded? Can it change for some reason? Like for managing different environments for the API. Just double checking.